src/ORM/Table.php

```diff
@@ -1436,6 +1435,36 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * Handles the logic executing of a worker inside a transaction.
+     *
+     * @param bool $atomic Whether to execute the worker inside a database transaction.
+     * @param callable $worker The worker that will run inside the transaction.
+     * @return mixed
+     */
+    protected function _executeTransaction($atomic, callable $worker)
```

Comment:
I prefer callable first but meh, this is a protected method anyway so I don't mind either way. Just would prefer if the method was renamed.